MSQL/MSQLTask2/5/a.sql

```diff
@@ -0,0 +1,14 @@
+USE [Northwind]
+GO
+
+SELECT *
+  FROM [dbo].[Customers]
+  Where  [Region] = 'SP' 
+
+INTERSECT
```

Comment:
EXCEPT should be used here. I saw it on the screenshot, but here in sql script you probably forgot to change.

Reply:
Maybe. I forgot make script and rewrited it letter